beard_server/app.py

```diff
@@ -40,4 +40,7 @@
 logging.basicConfig(stream=sys.stderr)
 
 if __name__ == '__main__':
+    # Text endpoint sends back keys UTF-8.
+    app.config['JSON_AS_ASCII'] = False
```

Comment:
@lnielsen, do you know how in Invenio you are doing for defaulting JSON responses in APIs to support UTF8?